packages/twenty-front/src/modules/activities/inline-cell/components/ActivityTargetsInlineCell.tsx

```diff
@@ -1,24 +1,41 @@
+import { Key } from 'ts-key-enum';
+
 import { ActivityTargetChips } from '@/activities/components/ActivityTargetChips';
 import { useActivityTargetObjectRecords } from '@/activities/hooks/useActivityTargetObjectRecords';
 import { ActivityTargetInlineCellEditMode } from '@/activities/inline-cell/components/ActivityTargetInlineCellEditMode';
 import { Activity } from '@/activities/types/Activity';
+import { ActivityEditorHotkeyScope } from '@/activities/types/ActivityEditorHotkeyScope';
 import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
 import { useFieldContext } from '@/object-record/hooks/useFieldContext';
 import { RecordFieldInputScope } from '@/object-record/record-field/scopes/RecordFieldInputScope';
 import { RecordInlineCellContainer } from '@/object-record/record-inline-cell/components/RecordInlineCellContainer';
-import { RelationPickerHotkeyScope } from '@/object-record/relation-picker/types/RelationPickerHotkeyScope';
+import { useInlineCell } from '@/object-record/record-inline-cell/hooks/useInlineCell';
 import { IconArrowUpRight, IconPencil } from '@/ui/display/icon';
+import { useScopedHotkeys } from '@/ui/utilities/hotkey/hooks/useScopedHotkeys';
 
 type ActivityTargetsInlineCellProps = {
+  onFocus?: () => void;
+  onBlur?: () => void;
   activity: Activity;
 };
 
 export const ActivityTargetsInlineCell = ({
   activity,
+  onFocus,
+  onBlur,
 }: ActivityTargetsInlineCellProps) => {
   const { activityTargetObjectRecords } = useActivityTargetObjectRecords({
     activityId: activity?.id ?? '',
   });
+  const { closeInlineCell } = useInlineCell();
```

Comment:
this is actually not needed, we should leave useInlineCell handle it (there was a bug as <FieldContextProvider> is too low in the component hiearchy making this useInlineCell out of context)